mmdet/models/dense_heads/fcos_head.py

```diff
@@ -33,6 +46,10 @@ def __init__(self,
                                  (512, INF)),
                  center_sampling=False,
                  center_sample_radius=1.5,
+                 norm_on_bbox=False,
+                 centerness_on_reg=False,
+                 dcn_on_last_conv=False,
+                 conv_bias=False,
```

Comment:
We may directly make `conv_bias`  accept the same options as that in [mmcv](https://github.com/open-mmlab/mmcv/blob/67a26da917c553561b93aa0fc75faedc70781027/mmcv/cnn/bricks/conv_module.py#L34)